packages/host/app/services/matrix-service.ts

```diff
@@ -534,8 +530,8 @@ export default class MatrixService extends Service {
   async addCardsToRoom(
     cards: CardDef[],
     roomId: string,
-    cardHashes: Map<string, string>,
-    opts?: CardAPI.SerializeOpts,
+    cardHashes: Map<string, string> = this.cardHashes,
+    opts: CardAPI.SerializeOpts = { maybeRelativeURL: null },
```

Comment:
Similarly we could replace this with whether we should include computeds, that's the only serialisation option we distinguish between. Not key for this PR but could be a good cleanup.